pkg/machinery/config/types/v1alpha1/v1alpha1_features.go

```diff
@@ -12,3 +12,12 @@ func (f *FeaturesConfig) RBACEnabled() bool {
 
 	return *f.RBAC
 }
+
+// StableHostnameEnabled implements config.Features interface.
+func (f *FeaturesConfig) StableHostnameEnabled() bool {
+	if f.StableHostname == nil {
```

Comment:
we have a shortcut for this: https://github.com/siderolabs/go-pointer/blob/master/pointer.go#L12-L21